templates/analytics/stats.html

```diff
@@ -10,6 +10,8 @@
 {{ minified_js('stats')|safe }}
 {% stylesheet 'stats' %}
 
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```

Comment:
Zulip ships all its dependencies, including bootstrap; maybe you're using a feature of bootstrap newer than what we have (which is some 2.x)?